securedrop/source_templates/logout.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block body %}
+<div class="">
+  <a href="{{ url_for('main.login') }}" class="btn pull-right"> {{ gettext('LOG IN') }}
+  </a>
+  <br>
+<div>
+  <br>
+  <h1>{{ gettext('One more thing...') }}</h1>
+  <p id="click-new-identity-tor"> {{ gettext('Click the <img src={icon} alt="broom icon" width="16" height="16">&nbsp;<strong>New Identity</strong> button in your Tor browser\'s toolbar. This will clear your Tor browser activity data on this device.').format(icon=url_for('static', filename='i/torbroom-black.png'))  }}</p>
+</div>
+</div>
+{% endblock %}
```

Comment:
Use of `<div>`s for logical grouping is partially a matter of taste, so leaving up to implementer and final reviewer how you want to group it, but just noting that the most compact version of this template is this:

```html
<a href="{{ url_for('main.login') }}" class="btn pull-right"> {{ gettext('LOG IN') }}</a>
<br class="clearfix">
<h1>{{ gettext('One more thing...') }}</h1>
<p id="click-new-identity-tor">...</p>
```

Regardless of how you want to employ `<div>`s, you can get rid of the `class=""` specification in line 3.

Reply:
oops- that was a remnant of something else, should not have made it in there. thank you for catching!

Reply:
fixed in 0acdd39

Reply:
updated in 0acdd39